### PR TITLE
H levels

### PR DESCRIPTION
**Pull request by IanRay11**
I needed to finish this proof, but you should probably review it.

### CODE REVIEW

**Comment by martinescardo** [comment]
https://github.com/martinescardo/TypeTopology/pull/236/conflicts

**Comment by IanRay11** [comment]
Oh shoot. I thought I was pushing the same PR

**Comment by IanRay11** [comment]
I need to get some sleep and come back to this, again sorry

**Comment by martinescardo** [comment]
No worries. Have a good rest.

**Comment by IanRay11** [comment]
You might want to review what I've added with the recent push @martinescardo

**Comment by martinescardo** [comment]
OK. I will do later.

File: source/UF/H-Levels.lagda [hunk 1]
```diff
@@ -153,11 +154,61 @@ hlevel-closed-under-Π {𝓤} (succ n) X Y m f g =
...
 \end{code}
+
+From Univalence we can show that (ℍ n) is of level (n + 1), for all n : ℕ.
+
+The next lemma should probably be added as it is the equivalent definition
+of is-prop.
+
+\begin{code}
+
+lemma : {X : 𝓤 ̇} → ((x x' : X) → is-contr (x ＝ x')) → is-prop X
```
Comment (martinescardo) [inline source/UF/H-Levels.lagda:173]: `lemma` is not a good name for such a general fact. How about `is-prop-criterion`. But if you have a better idea, go ahead. Please just avoid `lemma`.

File: source/UF/H-Levels.lagda [hunk 2]
```diff
@@ -153,11 +154,61 @@ hlevel-closed-under-Π {𝓤} (succ n) X Y m f g =
...
+lemma f x x' = center (f x x')
+
+ℍ-is-next-level : (n : ℕ)
```
Comment (martinescardo) [inline source/UF/H-Levels.lagda:176]: I would prefer `ℍ-is-of-next-hlevel` (two changes).

**Review by martinescardo — CHANGES_REQUESTED** [review]
Great. I required minimal notational changes only.

**Review by martinescardo — APPROVED** [review]

**Pull request merged**